pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -6,6 +6,7 @@
 , CoreServices, Cocoa, Hypervisor, rez, setfile, vmnet
 , guestAgentSupport ? (with stdenv.hostPlatform; isLinux || isNetBSD || isOpenBSD || isSunOS || isWindows) && !toolsOnly
 , numaSupport ? stdenv.isLinux && !stdenv.isAarch32 && !toolsOnly, numactl
+, waylandForwardingSupport ? stdenv.isLinux && !nixosTestRunner && !toolsOnly, rutabaga-gfx-ffi
```

Comment:
Rutabaga is supposed cross-platform.  Is there any reason to restrict this to Linux?

Reply:
Also let's call this "rutabagaSupport", because Rutabaga does a lot more than just Wayland forwarding.

Reply:
Afaik, it's only supported on Linux. I could potentially work on BSDs?

Reply:
It has macOS support IIRC.  I remember seeing some commits about it at least.

Reply:
Ok, but the qemu library seems to specifically use wayland on the host? Not sure how this is supposed to work on macOS.

Reply:
Me neither.  Maybe if you want to emulate a GPU but not have a window for it.

Reply:
I would leave this to someone, who can actually test the build on macOS. Especially the graphical part.